CRM/Financial/Form/Payment.php

```diff
@@ -117,7 +117,7 @@ public function setDefaultValues() {
    */
   public static function addCreditCardJs($paymentProcessorID = NULL, $region = 'billing-block') {
     $creditCards = CRM_Financial_BAO_PaymentProcessor::getCreditCards($paymentProcessorID);
-    $creditCardTypes = CRM_Core_Payment_Form::getCreditCardCSSNames($creditCards);
+    $creditCardTypes = CRM_Contribute_PseudoConstant::creditCard($creditCards);
```

Comment:
So we want to be assigning an array of name=>label yeah - ie

[  
  'visa' =.> 'Visa',
  'mastercard' => 'MasterCard'
/....
]